src/index.js

```diff
@@ -47,24 +47,23 @@ export function sampleRUM(checkpoint, data = {}) {
     if (!window.hlx.rum) {
       const usp = new URLSearchParams(window.location.search);
       const weight = (usp.get('rum') === 'on') ? 1 : 100; // with parameter, weight is 1. Defaults to 100.
-      // eslint-disable-next-line no-bitwise
-      const hashCode = (s) => s.split('').reduce((a, b) => (((a << 5) - a) + b.charCodeAt(0)) | 0, 0);
-      const id = `${hashCode(window.location.href)}-${new Date().getTime()}-${Math.random().toString(16).substr(2, 14)}`;
       const random = Math.random();
+      const id = Math.floor(random * 999999).toString().padStart(6, '0');
```

Comment:
I'd go without the `padStart` (anxious about browser support) and probably map each pair through https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/fromCharCode so that the new IDs are just three numbers or letters, instead of six number digits

Reply:
`fromCharCode` can return non-alphanumeric characters, which I think may not be the best to generate an ID that is sent over the network and persisted in the database.
E.g. 
```
String.fromCharCode(65,34,92) = A"\\
```

Would something like this work?
```
btoa(Math.floor(100000 + Math.random() * 900000)).substring(0,3)`
```

Reply:
We can push the values into the `a-zA-Z0-9` range of char codes to keep it readable. `btoa` may not have wide enough browser support.

Reply:
I'm not sure if I follow you. 

The code below would:
* generate a 6 digits random number
* split it in 3, 2-digits number
* force each of the 2 digits numbers into a valid alphanumeric character code 
* get the character codes, transform them in an alphanumeric char, and join them in a string

This works, but I think it is getting out of our hands.
```
const id = Math.floor(100000 + Math.random() * 900000).toString().match(/.{2}/g).map((n) => {
   const randomInt = parseInt(n);
   const {min, range} = randomInt < 34 ? {min: 48, range: 10} : (randomInt < 68 ? {min: 65, range: 26} : {min: 94, range: 26});
    return String.fromCharCode(Math.floor((randomInt * range) /  100 + min))})
.join('')
```

A simpler solution, which will generate a 3 chars id using numbers and lower letters.
```
const id = '00'.concat(Math.random().toString(36)).slice(-3); 
```
the `00` is needed in case of a random very close to 1, i.e. `0.999999999999999999999`

Reply:
A bit uglier approach:
```
const alphaChars = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
const id = [...Array(3)].reduce(c => c + alphaChars[Math.floor(Math.random()*62)],'');
```